AUTHORS

```diff
@@ -102,3 +102,4 @@ Yevgeniy Vahlis <evahlis@gmail.com>
 Yucheng Zhu <zyctc000@gmail.com>
 Zoe Juozapaitis
 Patrick DeVivo <patrick@tickgit.com>
+Mark Skilbeck <markskilbeck@gmail.com>
```

Comment:
I'll fix up the author above in another PR, but can you put your name in alphabetically, please.

Reply:
apologies

Reply:
There were counter examples in the file that would lead you astray, approved by me, no apologies necessary.